### PR TITLE
removed logo from navbar and footer

### PR DESCRIPTION
**Pull request by arpit456jain**
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)

### What does this implement/fix? Explain your changes.
removed logo from navbar and footer

### Does this close any currently open issues?
Fixes : #186

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <img alt="PHP" src="https://img.shields.io/badge/php-%23777BB4.svg?&style=for-the-badge&logo=php&logoColor=white"/> <img alt="JavaScript" src="https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E"/> <img alt="Python" src="https://img.shields.io/badge/python%20-%2314354C.svg?&style=for-the-badge&logo=python&logoColor=white"/> <img alt="CSS3" src="https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white"/>
 
-<h1><img  align="right;" alt="Brand Logo " src="https://github.com/siv2r/kidney-exchange/blob/master/images/logo3.png?raw=true" width="25" height="25">&nbsp;Kidney Exchange Program</h1>
+<h1>&nbsp;Kidney Exchange Program</h1>
```
Comment (siv2r) [inline README.md:3]: what is `&nbsp`?

Comment (arpit456jain) [reply README.md:3]: i think its for space 
its already there should i remove it too ?

**Comment by arpit456jain** [comment]
kid

File: README.md (hunk 1 above)
Comment (siv2r) [reply README.md:3]: Yes, you can remove it. Also, you can use `#` instead of the `h1` tag for the heading

**Comment by arpit456jain** [comment]
kid
screencapture-localhost-kidney-exchange-2021-05-27-22_53_11

**Comment by arpit456jain** [comment]
don't worry about footer my next issue is footer

**Comment by arpit456jain** [comment]
will align github logo in center and decrease its width

**Comment by arpit456jain** [comment]
screencapture-localhost-kidney-exchange-2021-05-27-22_53_11

**Pull request merged**